frontend/components/panels/EmployeePanel.scss

```diff
@@ -1,12 +1,12 @@
 /*
- * This file is part of Search NEU and licensed under AGPL3. 
- * See the license file in the root folder for details. 
+ * This file is part of Search NEU and licensed under AGPL3.
+ * See the license file in the root folder for details.
  */
 
- .header {
+.page-home .header {
```

Comment:
FYI this selector is matching all the panel headers on the page (the class panel headers and the employee panel headers) rn

Reply:
I just compared it to live, this seems correct?

Reply:
In live, however, we were using css modules so every css file only effected the JS file where it was used. The css selectors themselves were transformed into something else before they were injected into the page. `.classTitle` could be transformed into `.frontend-components-panels-___DesktopClassPanel__classTitle___2pT1j`, which would prevent any css in each file from affecting anything outside of where it was used. 

All of DesktopClassPanel.css could look something like this: https://gist.github.com/ryanhugh/9fad5bc0978e8e5df163adcd51816f3f

Reply:
Ah, that explains why sass styles were being applied globally. From now on, it looks like we're going to need to properly nest our styles. Should be easy, since we're using sass!